client/my-sites/people/people-profile/index.jsx

```diff
@@ -80,6 +81,8 @@ const PeopleProfile = React.createClass( {
 		return 'role-' + role;
 	},
 
+	handleLinkToReaderSiteStream: () => recordTrack( 'calypso_reader_people_followers_link_click' ),
```

Comment:
hrm. this isn't really a reader event. probably `calypso_sites_people_followers_link_click` instead?

Reply:
sure